lms/djangoapps/grades/api/v1/gradebook_views.py

```diff
@@ -663,9 +666,10 @@ def post(self, request, course_key):
             requested_user_id = user_data['user_id']
             requested_usage_id = user_data['usage_id']
             try:
-                user = self._get_single_user(request, course_key, requested_user_id)
+                user = self._get_single_user(request.user, course_key, requested_user_id)
                 usage_key = UsageKey.from_string(requested_usage_id)
             except (USER_MODEL.DoesNotExist, InvalidKeyError, CourseEnrollment.DoesNotExist) as exc:
+                self._log_update_result(request.user, requested_user_id, requested_usage_id, None, None, False)
```

Comment:
If you make the default parameter changes I talk about below, you can call this as:
```python
self._log_update_result(request.user, requested_user_id, requested_usage_id, success=False)
```